app/controllers/concerns/hyrax/works_controller_behavior.rb

```diff
@@ -68,6 +68,8 @@ def create
     # Finds a solr document matching the id and sets @presenter
     # @raise CanCan::AccessDenied if the document is not found or the user doesn't have access to it.
     def show
+      @user_collections = collections_service.search_results(:deposit)
```

Comment:
How about guarding this line so it doesn't run for every work show request since it looks like it only gets used `if presenter.editor?`